utils/test_utils/src/chain_structures.rs

```diff
@@ -126,13 +128,20 @@ pub fn construct_full_tipset() -> FullTipset {
     FullTipset::new(blocks).unwrap()
 }
 
-const DUMMY_SIG: [u8; 1] = [0u8];
+pub const DUMMY_SIG: [u8; 1] = [0u8];
 
-struct DummySigner;
-impl Signer for DummySigner {
-    fn sign_bytes(&self, _: &[u8], _: &Address) -> Result<Signature, anyhow::Error> {
+pub struct DummySigner;
+
+impl DummySigner {
+    pub fn sign_bytes(_: &[u8], _: &Address) -> Result<Signature> {
         Ok(Signature::new_secp256k1(DUMMY_SIG.to_vec()))
     }
+
+    pub fn sign_message(message: Message) -> Result<SignedMessage> {
+        let bz = message.cid()?.to_bytes();
+        let signature = Self::sign_bytes(&bz, &message.from)?;
+        Ok(SignedMessage::new_unchecked(message, signature))
+    }
```

Comment:
Creating a fake signature can never fail.

```suggestion
    pub fn sign_message(message: Message) -> SignedMessage {
        let signature = Signature::new_secp256k1(DUMMY_SIG.to_vec());
        SignedMessage::new_unchecked(message, signature)
    }
```

Reply:
Have removed `DummySigner`